docs/setup/docker.asciidoc

```diff
@@ -94,7 +94,7 @@ Some example translations are shown here:
 **Environment Variable**:: **Kibana Setting**
```

Comment:
I found on other doc that includes xpack.monitoring:

docs/management/managing-licenses.asciidoc
109  `xpack.monitoring.history.duration`